Denik CZ.js

```diff
@@ -0,0 +1,110 @@
+function detectWeb(doc, url) {
+		return "newspaperArticle";
+}
+
+function doWeb(doc, url) {
+	var arts = new Array();
+	scrape(doc);
+}
+
+function scrape(doc) {
+	var newArticle = new Zotero.Item('newspaperArticle');
+	newArticle.url = doc.location.href;
+	newArticle.title = ZU.trimInternal(ZU.xpathText(doc, '//div[@class = "dv3-clanek-left-fix"]/h1')).replace(/^,/, "");
+	var date = ZU.xpathText(doc, '(//div[@class="dv3-clanek-datum"])');
+	var matched = ZU.trimInternal(date).match(/dnes/)
+	if (matched) {
+		var actualDate = new Date();
+		var actualDateStr = '';
+		actualDateStr = actualDate.getUTCFullYear() + '-' +
+    	('00' + (actualDate.getUTCMonth()+1)).slice(-2) + '-' +
+    	('00' + actualDate.getUTCDate()).slice(-2);
+    	date = actualDateStr;
+	} else {
+		var parts = ZU.trimInternal(date).split('.');
+		
+		var actualDate = new Date(parts[2],parts[1]-1, parts[0]);
+		var actualDateStr = '';
+		actualDateStr = actualDate.getUTCFullYear() + '-' +
+    	('00' + (actualDate.getUTCMonth()+1)).slice(-2) + '-' +
+    	('00' + actualDate.getDate()).slice(-2);
+	}
+	//Zotero.debug(ZU.trimInternal(date).match('16'));
+	if (actualDateStr) newArticle.date = actualDateStr;
+	var teaser = ZU.xpathText(doc, '//p[@class="dv3-clanek-perex"]');
+	if (teaser != null) {
+		newArticle.abstractNote = Zotero.Utilities.trimInternal(teaser).replace(/^,\s*/, "");
+	}
+
+	//some authors are in /a, some aren't we need to distinguish to get this right
+	if (ZU.xpathText(doc, '//p[@class = "clanek-autor"]/a') != null) {
+		var xpath = '//p[@class = "clanek-autor"]/a';
+	} else {
+		var xpath = '//p[@class = "clanek-autor"]';
```

Comment:
I don't think this case would work the same way, since it includes "Autor: ". Unless there's a good reason to do otherwise, I would say grab the contents of the `p` in either case, trim off the "Autor: " and parse the author. FYI, some authors are in the format ["Redakce / ČTK"](http://www.denik.cz/auto/znacky-na-prani-v-cesku-bude-jezdit-hvezdna-lod-enterprise-jagr0068-neprosel-20160108.html).